pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -132,3 +134,15 @@ def makevector(*x):
         return torch.tensor(x, dtype=torch_dtype)
 
     return makevector
+
+
+@pytorch_funcify.register(OpFromGraph)
+def pytorch_funcify_OpFromGraph(op, node=None, **kwargs):
+    _ = kwargs.pop("storage_map", None)
+
+    fgraph_fn = torch.compile(pytorch_funcify(op.fgraph, **kwargs))
+    return (
+        fgraph_fn
+        if len(op.fgraph.outputs) > 1
+        else lambda *args: itemgetter(0)(fgraph_fn(*args))
+    )
```

Comment:
It's nicer to pass a function, because it's more readable when inspecting the the generated source code for the compiled function